lib/util.js

```diff
@@ -291,6 +309,65 @@ exports.getVideoDescription = function(html) {
 
 
 /**
+ * Get video Owner from html.
+ *
+ * @param {String} body
+ * @return {Object}
+ */
+var authorRegexp = /<a href="\/channel\/([\w-]+)"[^>]+>(.+?(?=<\/a>))/
+var aliasRegExp = /<a href="\/user\/([^"]+)/
+exports.getAuthor = function(body) {
+  var ownerinfo = this.between(body, '<div id="watch7-user-header" class=" spf-link ">', '<div id="watch8-action-buttons" class="watch-action-buttons clearfix">');
+  ownerinfo = new Entities().decode(ownerinfo)
+  if (ownerinfo === '') {
+	  return {};
+  }
+  let match = ownerinfo.match(authorRegexp);
+  return {
+	'type':'channel',
+	'ref':'/channel/'+match[1],
+	'id':match[1],
+	'name':match[2],
+	'avatar':this.between(ownerinfo, 'data-thumb="','"'),
+	'aka': (ownerinfo.includes('/user/') ? '/user/'+ownerinfo.match(aliasRegExp)[1] : 'none'),
+  };
+};
+
+
+/**
+ * Get video published at from html.
+ *
+ * @param {String} body
+ * @return {String}
+ */
+exports.getPublished = function(body) {
+  return Date.parse(this.between(body, '<meta itemprop="datePublished" content="', '">'))
+};
+
+
+/**
+ * Get video published at from html.
+ * Credits to https://github.com/paixaop.
+ *
+ * @param {String} body
+ * @return {Array.<Object>}
+ */
+exports.getRelatedVideos = function(body) {
+  var jsonStr = this.between(body, '\'RELATED_PLAYER_ARGS\': {"rvs":','},')
+  try {
+    jsonStr = JSON.parse(jsonStr)
+  }
+  catch(err) {
+	return [];
+  }
+  var unparsed_videos = jsonStr.split(',')
+  return unparsed_videos.map(function(link) {
```

Comment:
Since it the `unparsed_videos` variable will only be used once and in the line right after, you can put `jsonStr.split(',')` here instead.